### PR TITLE
SL-3780 Create versioned comments connection

### PR DESCRIPTION
**Pull request by sergioebx**
### Description of Changes
- Create versioned comments connection class
- Copy `postComment` method from v2 to versioned comments connection class
- Tag v2 comment class as deprecated

### Testing
Manual testing

### CODE REVIEW

File: src/main/java/com/echobox/api/linkedin/connection/versioned/VersionedCommentsConnection.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import com.echobox.api.linkedin.types.social.actions.CommentResponse;
+import com.echobox.api.linkedin.types.urn.URN;
+
+/**
+ * Comment connection class for posting comments
+ *
+ * @author Sergio Abplanalp
+ *
+ */
+public class VersionedCommentsConnection extends VersionedConnection {
```
Comment (kennethwong-hc) [inline src/main/java/com/echobox/api/linkedin/connection/versioned/VersionedCommentsConnection.java:31]:
```suggestion
public class VersionedCommentConnection extends VersionedConnection {
```
It seems we are using singular form for connection classes (except AssetsConnection, which will be deprecated soon)

**Review by kennethwong-hc — COMMENTED** [review]
Looks good. Just one minor point on naming.
Did you perform a manual test?

**Comment by sergioebx** [comment]
> Looks good. Just one minor point on naming. Did you perform a manual test?

I've tested this manually

**Review by kennethwong-hc — APPROVED** [review]
Thanks

**Pull request merged**